core/src/utils/animation/animation-utils.ts

```diff
@@ -1,10 +1,26 @@
+export const getAnimationPrefix = (el: HTMLElement): string => {
+  const supportsUnprefixed = (el.style as any).animationName !== undefined;
+  const supportsWebkitPrefix = (el.style as any).webkitAnimationName !== undefined;
+
+  if (!supportsUnprefixed && supportsWebkitPrefix) {
+    return '-webkit-';
+  }
+
+  return '';
+};
 
 export const setStyleProperty = (element: HTMLElement, propertyName: string, value: string | null) => {
+  if (propertyName.startsWith('animation')) {
+    propertyName = getAnimationPrefix(element) + propertyName;
+  }
   element.style.setProperty(propertyName, value);
 };
 
 export const removeStyleProperty = (element: HTMLElement, propertyName: string) => {
-  element.style.removeProperty(propertyName);
+  if (propertyName.startsWith('animation')) {
+    propertyName = getAnimationPrefix(element) + propertyName;
+  }
+  element.style.removeProperty(getAnimationPrefix(element) + propertyName);
```

Comment:
This looks like it will prepend the vendor prefix twice.

Test case:
`propertyName = 'animation-name'`

`if (propertyName.startsWith('animation')) {` returns true, so `propertyName = '-webkit-animation-name'.

And then you remove the property by doing `'-webkit-' + propertyName`, which will end up being `'-webkit--webkit-animation-name'`.



Might just be easiest to do something like:

```
const prefix = propertyName.startsWith('animation') ? getAnimationPrefix(element) : '';
element.style.removeProperty(`${prefix}${propertyName}`);
```
```

Reply:
Oh I'm really sorry to waste your time. I forgot to remove that.

Reply:
No worries - thanks!